src/_css/UserDetailComponent.scss

```diff
@@ -22,3 +89,6 @@ h3.ui.header {
   width: 90%;
   align-content: center;
 }
+
+
```

Comment:
Please remove the 2 extra spaces

Reply:
You have not removed the extra spaces. You can configure your IDE to automatically remove trailing white spaces